### ISSUE TITLE
Option to deactivate "Remember me" globally

### ISSUE THREAD

**Issue by widhalmt** [Icinga/icingaweb2#4506]
We have a customer who has a security policy in place that, how ever, renders any "Remember Me" function of web interfaces useless. When an user activates remember me, they are locked out until they manually delete cookies.

Please don't ask me about the specifics, they are hidden by their security team.

But since we won't be able to circumvent that policy we need a way to disable "Remember Me" completely. I'm ok with just the button not being shown to users as long as that will also keep any maintenance jobs etc from trying to delete things that can't be there.

Of course it could be sufficient to tell users "just don't use that feature" but the problem is we have lots of setups where there are many users working with Icinga Web 2 and relying on them all getting this information and sticking to it might not always be feasible.

ref/NC/730835

**Comment by lippserd** [comment]
> We have a customer who has a security policy in place that, how ever, renders any "Remember Me" function of web interfaces useless. When an user activates remember me, they are locked out until they manually delete cookies.

How does this work? Because we already have cookies.

**Comment by widhalmt** [comment]
To be honest - I don't know, how it works. I can ask for specifics. I'll keep you posted.

**Comment by lippserd** [comment]
> To be honest - I don't know, how it works. I can ask for specifics. I'll keep you posted.

Please reopen this issue then. I don't think it's right to leave the feature open without even knowing if it's actually needed. I am also strictly against such a configuration option.

@nilmerg Feel free to close.

**Comment by widhalmt** [comment]
We had some discussion about this offline. I opened this issue partly to mitigate whatever obscure security policies might give problems with it (I assumed that we only just found the first of more setups with problems to come) and partly because I'm a fan of making as many features optional as possible.

In the offline discussions you convinced me that it's not a good idea, or at least not worth the hassle to have optionality implemented just for the sake of "feeling right". As for the security policy part - I'm waiting for details and when I get some, I'll come back to discuss them with you and *might* reopen the issue then. For now I'll be happy to close it.
